safe_transaction_service/history/signals.py

```diff
@@ -154,19 +156,22 @@ def process_webhook(
     for payload in payloads:
         if address := payload.get("address"):
             if is_relevant_notification(sender, instance, created):
-                logger.debug(
-                    "Triggering send_webhook and send_notification tasks for created=%s object=%s",
-                    created,
-                    instance,
-                )
-                send_webhook_task.apply_async(
-                    args=(address, payload), priority=2  # Almost lowest priority
-                )  # Almost the lowest priority
-                send_notification_task.apply_async(
-                    args=(address, payload),
-                    countdown=5,
-                    priority=2,  # Almost lowest priority
-                )
+                if mark_notification_as_processed(address, payload):
```

Comment:
@Uxio0 I move the check if payload is repeated here, at this way we could avoid repeated webhooks and repeated notifications, but maybe is in notifications because some reason that I missing.

Reply:
I think it's a good idea but I would not change it on this PR, as it can break the current behaviour. But if you already refactored the tests don't worry and we will merge it like this

Reply:
I did not change the tests, but the tests still works. 
I'm concerned about repeated events in the queue.
If we don't want to change the code here, maybe I can call `is_duplicated` method before to call `send_notification` (because send_notification is setting the key) If I do this after `send_notification`  then the event will be send never.

Reply:
Anyway, we can handle it in a different PR. 
@Uxio0 I removed the commit that included the mentioned changes.